server/src/test/java/org/elasticsearch/transport/RemoteClusterServiceTests.java

```diff
@@ -911,104 +915,32 @@ public void testReconnectWhenSeedsNodesOrProxyAreUpdated() throws Exception {
-                    service.updateRemoteCluster(
-                        "cluster_test",
-                        newSeeds, node1.getAddress().toString(),
-                        genericProfile("cluster_test"), connectionListener(thirdLatch));
-                    thirdLatch.await();
-
-                    assertBusy(() -> {
-                        assertFalse(secondRemoteClusterConnection.isNodeConnected(node0));
-                        assertFalse(secondRemoteClusterConnection.isNodeConnected(node1));
-                        assertEquals(0, secondRemoteClusterConnection.getNumNodesConnected());
-                        assertTrue(secondRemoteClusterConnection.isClosed());
-                    });
-
-                    final RemoteClusterConnection thirdRemoteClusterConnection = service.getRemoteClusterConnection("cluster_test");
-                    assertTrue(thirdRemoteClusterConnection.isNodeConnected(node1));
-                    // Will only successfully connect to node1 because the proxy address is to node1 and
-                    // validation will fail when attempt to connect to node0
-                    assertFalse(thirdRemoteClusterConnection.isNodeConnected(node0));
-                    assertEquals(1, thirdRemoteClusterConnection.getNumNodesConnected());
-                    assertFalse(thirdRemoteClusterConnection.isClosed());
+//                    final CountDownLatch thirdLatch = new CountDownLatch(1);
+//                    service.updateRemoteCluster(
+//                        "cluster_test",
+//                        createSettings(newSeeds), node1.getAddress().toString(),
+//                        genericProfile("cluster_test"), connectionListener(thirdLatch));
+//                    thirdLatch.await();
+//
+//                    assertBusy(() -> {
+//                        assertFalse(secondRemoteClusterConnection.isNodeConnected(node0));
+//                        assertFalse(secondRemoteClusterConnection.isNodeConnected(node1));
+//                        assertEquals(0, secondRemoteClusterConnection.getNumNodesConnected());
+//                        assertTrue(secondRemoteClusterConnection.isClosed());
+//                    });
+//
+//                    final RemoteClusterConnection thirdRemoteClusterConnection = service.getRemoteClusterConnection("cluster_test");
+//                    assertTrue(thirdRemoteClusterConnection.isNodeConnected(node1));
+//                    // Will only successfully connect to node1 because the proxy address is to node1 and
+//                    // validation will fail when attempt to connect to node0
+//                    assertFalse(thirdRemoteClusterConnection.isNodeConnected(node0));
+//                    assertEquals(1, thirdRemoteClusterConnection.getNumNodesConnected());
+//                    assertFalse(thirdRemoteClusterConnection.isClosed());
                 }
             }
         }
     }
 
-    public void testRemoteClusterWithProxy() throws Exception {
```

Comment:
where was this test moved?

Reply:
`SniffConnectionStrategyTests#testConfiguredProxyAddressModeWillReplaceNodeAddress` in a previous PR.